src/entities/purchase-params.ts

```diff
@@ -21,4 +21,26 @@ export interface PurchaseParams {
    * The email of the user. If undefined, RevenueCat will ask the customer for their email.
    */
   customerEmail?: string;
+
+  /**
+   * Optional customization for the translations used in the purchase flow.
+   * The keys are the locale codes and the values are objects with the same keys as the default translations.
+   */
+  customTranslations?: Record<string, Record<string, string>>;
```

Comment:
this type is repeated in a few places. we can make a definition for this

Reply:
Created the CustomTranslations type